sdcm/sct_runner.py

```diff
@@ -27,7 +27,7 @@ class ImageType(Enum):
 
 class SctRunner:
     """Provisions and configures the SCT runner"""
-    VERSION = 1.2  # Version of the Image
+    VERSION = 1.4  # Version of the Image
```

Comment:
Curious why it is not 1.3
But not a blocker for me.

Reply:
We have image with 1.3 already created in eu-west-2 region (not sure why), so I wanted to avoid overwriting that :)

Reply:
@bentsi , is it ok? ^

Reply:
yes